### ISSUE TITLE
osgearth x86-windows dependencies

### ISSUE THREAD

**Issue by stevegbarry** [gwaldron/osgearth#1603]
Using vcpkg to build osgearth for x86 errors on a dependence on nvtt - stating x86 architecture is not supported by nvtt.  Is there a workaround to be able to build osgearth for x86?

**Comment by gwaldron** [comment]
osgEarth does not depend on NVTT -- are you using it elsewhere?

**Comment by stevegbarry** [comment]
I am not using it elsewhere - so I do not know why it is in the build chain.  Below is the output from vcpkg.  Everything builds correctly using x64.

>vcpkg install osgearth:x86-windows
Computing installation plan...
The following packages will be built and installed:
  * nvtt[core]:x86-windows
  * osg[core,plugins]:x86-windows
    osgearth[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x86-windows...
Starting package 1/3: nvtt:x86-windows
Building package nvtt[core]:x86-windows...
Could not locate cached archive: 770b711b12a6eee038e2814204ad5d22e8588627.zip
CMake Error at scripts/cmake/vcpkg_fail_port_install.cmake:94 (message):
  Architecture 'x86' not supported by nvtt!

Call Stack (most recent call first):
  ports/nvtt/portfile.cmake:1 (vcpkg_fail_port_install)
  scripts/ports.cmake:136 (include)

**Comment by remoe** [comment]
@stevegbarry , don't know, but nvtt is in osg plugins:

https://github.com/microsoft/vcpkg/blob/master/ports/osg/CONTROL#L22

**Comment by gwaldron** [comment]
@stevegbarry I recommend you being this up with the vcpkg team; I've found them to be very responsive. They can probably address the problem. NVTT is indeed an optional dep of OpenSceneGraph but it's not necessary.

**Comment by plevy** [comment]
@stevegbarry I ran into something similar yesterday.  I update vcpkg frequently enough that I have noticed the Openscenegraph plugin fails to build occasionally on some nvtt updates.  Yesterday it was in nvsquish.  When that happens, I usually vcpkg remove nvtt.  I haven't tried x86 but I do know it has been problematic in x64 so sometimes I just leave it in vcpkg and remove it from the Openscenegraph cmake in hopes the next nvtt update will sort it out.
